### PR TITLE
Show more registers data

### PR DESCRIPTION
**Pull request by chr1sdi**
https://trello.com/c/Cet7azEm/371-food-demo-should-show-all-the-data

### CODE REVIEW

File: food_ratings/frontend/views.py [hunk 1]
```diff
@@ -56,7 +56,6 @@ def _entry(register, key):
     response = _get('%s/record/%s.json' % (url, key))
     return response.json()
 
-
```
Comment (philandstuff) [inline food_ratings/frontend/views.py:59]: you didn't write about removing this empty line in your commit message 😝

**Comment by philandstuff** [comment]
looks good 👍

File: food_ratings/frontend/views.py (hunk 1 above)
Comment (chr1sdi) [reply food_ratings/frontend/views.py:59]: Ending life of that worthless empty line wasn't worthy drawing your attention. As it's ironically happened let me express my highest level of disappointment as well as respect for the line that it has remained over there for so long. To put it this way: **NOT ON MY WATCH!**

Comment (philandstuff) [reply food_ratings/frontend/views.py:59]: although now I think about it I think two blank lines is [PEP8 style](https://www.python.org/dev/peps/pep-0008/#blank-lines) 😆

**Pull request merged**